service/lib/agama/dbus/manager_service.rb

```diff
@@ -71,11 +71,11 @@ def initialize(config, logger = nil)
       # @note The service runs its startup phase
       def start
         setup_cockpit
+        export
         # We need locale for data from users
         locale_client = Clients::Locale.new
         # TODO: test if we need to pass block with additional actions
         @ui_locale = UILocale.new(locale_client)
-        export
```

Comment:
interesting that it helps. I actually would expect issues in other way. When you export service and then localize it, so I see small window when you can get unlocalized texts.